scheduler/src/cook/mesos/api.clj

```diff
@@ -474,6 +475,10 @@
                                   :constraint/pattern pattern}]))
                             constraints)
         container (if (nil? container) [] (build-container user db-id container))
+        executor (when executor
+                   (case executor ;; will throw an exception if invalid value is provided
```

Comment:
This would be a pretty rare occurrence, right? If the job has a bad value the schema validation should fail.